### DISCUSSION TITLE
copie stronghold

### DISCUSSION THREAD

**Discussion by Mechardo** [iotaledger/firefly#7323] [Help & Support]
Hello, I have updated and it does not make a stronghold copy for me, the little circle remains thinking and spinning and it never ends, thank you!

**Comment by begonaalvarezd** [comment]
Hi, which version of Firefly are you using? For some profiles, in firefly iota, it might take a long time exporting a new backup

**Reply by Mechardo** [reply]
How long is it? Because before it was less than a minute and now you spend a lot of time thinking, the truth is, I haven't let it finish because it seemed strange to me that you spend 5 minutes and do nothing, thank you! 1.7.3 is the version

**Reply by HankMoody089** [reply]
Got the same issue and for me backup worked with waiting about 4-5 minutes...

**Reply by CrashOverride-lab** [reply]
I can confirm this. I have the same Error. 
Errorlog is empty.
OS. WIndows 11 Pro

Version 1.7.3
Sprache: de
Währung: USD
Node-Auswahl: Manuell
Plattform: win32
Plattformversion: 10.0.22621
Plattform-Architektur: x64
CPU-Anzahl: 24
Gesamtspeicher: 32488.7 MB
Freier Speicher: 16655.9 MB

**Reply by CrashOverride-lab** [reply]
Another Hint. On FIrefly Shimmer there are no such issues. This Error appears also on Stronghold Update and if someone trys to do it manually in the Profil with the Backup Option.

**Reply by Mechardo** [reply]
I'll leave it longer then, thanks!

**Reply by Mechardo** [reply]
correct, it took me about 8 minutes, but since it was configured to close after 5 minutes, the application has been blocked until the pc, I had to set it to 1 hour, which was the maximum and that is when it has been saved correctly The 8 minutes, I'm commenting so that you can put more time in the firefly application, otherwise it won't have time to create itself, greetings to all!

**Comment by Mechardo** [comment]
correct, it took me about 8 minutes, but since it was configured to close after 5 minutes, the application has been blocked until the pc, I had to set it to 1 hour, which was the maximum and that is when it has been saved correctly The 8 minutes, I'm commenting so that you can put more time in the firefly application, otherwise it won't have time to create itself, greetings to all!

**Comment by begonaalvarezd** [comment]
Hi, we just released the new version 1.7.4 which contains the fix

### Firefly IOTA 1.7.4 announcement message

We just released Firefly IOTA 1.7.4. Among the changes, it contains the fixes for macOS crash at startup and the slow stronghold export. 

:warning: Important: macOS 10.13 will not be supported anymore in Firefly IOTA, so if you are running macOS 10.13 we highly recommend upgrading your macOS version to a newer one. 

:pencil: Changelog:

- fix slow stronghold export
- fix macOS support (official drop support macOS <= 10.13)
- add official asmb staking event 6 results

:arrow_down:  Download:
https://firefly.iota.org/

:bug:  Bug Reporting:
https://github.com/iotaledger/firefly/issues/new?assignees=&labels=bug+report&template=bug_report.yml&title=%5BBug+report%5D%3A

----

Please reopen discussion if the issue persists 🙏🏼 happy week 🌷
